entitybroker/rest/src/java/org/sakaiproject/entitybroker/rest/EntityActionsManager.java

```diff
@@ -227,6 +230,7 @@ public ActionReturn handleCustomActionExecution(ActionsExecutable actionProvider
                     } else if (e.getCause().getClass().isAssignableFrom(SecurityException.class)) {
                         throw new SecurityException(e.getCause().getMessage() + " (rethrown)", e.getCause());
                     }
+                    log.error("Exception {} not rethrown, throwing RuntimeException.", e.getCause().getMessage());
```

Comment:
```suggestion
                    log.error("Target exception class not identified, re throwing as a RuntimeException: {}", e.toString());
```

Reply:
minor suggestion where e.toString() will also include the class name in the message which can be very useful in these situations!

Reply:
`toString()` is simply more informative than `getMessage()` and really should be used almost always when logging the reason

```
    public String toString() {
        String s = getClass().getName();
        String message = getLocalizedMessage();
        return (message != null) ? (s + ": " + message) : s;
    }
```